Makefile

```diff
@@ -63,15 +71,25 @@ lib/libbase64.o: $(OBJS)
 	$(OBJCOPY) --keep-global-symbols=lib/exports.txt $@
 
 lib/config.h:
-	@echo "#define HAVE_AVX2   $(HAVE_AVX2)"    > $@
-	@echo "#define HAVE_NEON32 $(HAVE_NEON32)" >> $@
-	@echo "#define HAVE_NEON64 $(HAVE_NEON64)" >> $@
-	@echo "#define HAVE_SSSE3  $(HAVE_SSSE3)"  >> $@
-	@echo "#define HAVE_SSE41  $(HAVE_SSE41)"  >> $@
-	@echo "#define HAVE_SSE42  $(HAVE_SSE42)"  >> $@
-	@echo "#define HAVE_AVX    $(HAVE_AVX)"    >> $@
+	@echo "#define HAVE_AVX2                  $(HAVE_AVX2)"                   > $@
+	@echo "#define HAVE_NEON32                $(HAVE_NEON32)"                >> $@
+	@echo "#define HAVE_NEON64                $(HAVE_NEON64)"                >> $@
+	@echo "#define HAVE_SSSE3                 $(HAVE_SSSE3)"                 >> $@
+	@echo "#define HAVE_SSE41                 $(HAVE_SSE41)"                 >> $@
+	@echo "#define HAVE_SSE42                 $(HAVE_SSE42)"                 >> $@
+	@echo "#define HAVE_AVX                   $(HAVE_AVX)"                   >> $@
+	@echo "#define HAVE_FAST_UNALIGNED_ACCESS $(HAVE_FAST_UNALIGNED_ACCESS)" >> $@
+
+lib/table_generator: lib/table_generator.o
+	$(CC) $(CFLAGS) -o $@ $^
+
+lib/tables.h: lib/table_generator.c
+	@$(MAKE) -f $(THIS_FILE) lib/table_generator
+	./lib/table_generator > $@
+
```

Comment:
Since `lib/tables.h` is now under source control, I wanted to generate a new one only if `lib/table_generator.c` was modified. This can be modified to overwrite `lib/tables.h` "each" time libbase64 is built as you suggest.